src/scss/global.scss

```diff
@@ -342,7 +342,7 @@ textarea {
   overflow: auto;
 }
 
-.profile-icon {
+.Participant__profile-icon {
```

Comment:
This is a red flag.

Classes from `global.scss` should be deleted so they don't interfere with proper styles of the components, not set component styles directly inside the global.

Reply:
removed 5f42a2b